20-packages.ks

```diff
@@ -94,17 +94,10 @@ system-storage-manager
 -dmraid
 -lvm2
 
-# grub
--freetype
--grubby
--os-prober
-
-# selinux toolchain of policycoreutils, libsemanage, ustr
+# selinux toolchain of policycoreutils, libsemanage (libselinux is needed tho)
 -policycoreutils
 -checkpolicy
 -selinux-policy*
--libselinux-python
--libselinux
 
 # Things it would be nice to loose
 -fedora-logos
```

Comment:
Yum started to complain recently without this change, likely an update in CentOS.